src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs

```diff
@@ -101,19 +101,29 @@ internal DefaultTestHostManager(Architecture architecture, Framework framework,
         private Action<object> ExitCallBack => (process) =>
         {
             var exitCode = 0;
+            var testHostProcessStdErrorStr = this.testHostProcessStdError.ToString();
```

Comment:
This logic is duplicated across Default and DotnetTestHostManagers. Should extract?